django/contrib/auth/tests/custom_user.py

```diff
@@ -221,6 +223,11 @@ class CustomUserBadRequiredFields(AbstractBaseUser):
     class Meta:
         app_label = 'auth'
 
+
+class CustomUserUUIDasPK(AbstractUser):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
```

Comment:
`uuid.uuid4` is a function, and, as understand, you want to set some random data for it, so you must call it.
But I don't think that default is needed there though.